jenkins-scripts/lib/colcon-default-devel-windows.bat

```diff
@@ -138,13 +139,22 @@ call %win_lib% :list_workspace_pkgs || goto :error
 echo # END SECTION
 
 :: Check if package is in colcon workspace
-echo # BEGIN SECTION: Update package !COLCON_PACKAGE! from gz to ignition
+echo # BEGIN SECTION: Check if package !COLCON_PACKAGE! is in colcon workspace
 echo Packages in workspace:
 colcon list --names-only
 
 colcon list --names-only | find "!COLCON_PACKAGE!" > nul 2>&1
 if errorlevel 1 (
+  echo # END SECTION
+  echo # BEGIN SECTION: try package name !COLCON_PACKAGE_ORIGINAL! without major version
+  set COLCON_PACKAGE=!COLCON_PACKAGE_ORIGINAL! > nul 2>&1

```

Comment:
I think the `> nul 2>&1` at the end of the line is appending a single ` ` to the end of `COLCON_PACKAGE`:

~~~
Update package gz-sim  from gz to ignition
~~~